src/vm/dwbucketmanager.hpp

```diff
@@ -1062,37 +1062,45 @@ void BaseBucketParamsManager::FindFaultingMethodInfo()
         if (throwable != NULL)
         {
             StackTraceArray traceData;
-            EXCEPTIONREF(throwable)->GetStackTrace(traceData);
-
+            ZeroMemory(&traceData, sizeof(traceData));
```

Comment:
The `IsCodeContractsFrame` condition above is always going to be false. `__ContractsRuntime` that it checks for does not exist anymore; and code contracts are deprecated as a whole in .NET Core. It would be best to delete this.

Reply:
Delete the whole method then as dead code?

Reply:
Also removed a chunk from the caller; since the method would do nothing.